test/cli/content_blocking_test.go

```diff
@@ -168,18 +193,18 @@ func TestContentBlocking(t *testing.T) {
 			})
 		}
 
-		// Confirm that denylist is active for every content path in 'testCases'
-		gwTestName := fmt.Sprintf("Gateway denies %s", testCase.name)
-		t.Run(gwTestName, func(t *testing.T) {
-			resp := client.Get(testCase.path)
-			assert.Equal(t, http.StatusGone, resp.StatusCode, statusExpl)
-			assert.Contains(t, resp.Body, blockedMsg, bodyExpl)
-		})
-
+		for _, format := range []string{"", "raw", "car"} { // CARs should also fail immediately since we access the file directly.
+			// Confirm that denylist is active for every content path in 'testCases' with different formats
+			gwTestName := fmt.Sprintf("Gateway denies %s with format %s", testCase.name, format)
+			t.Run(gwTestName, func(t *testing.T) {
+				resp := client.Get(testCase.path + "?format=" + format)
+				assert.Equal(t, http.StatusGone, resp.StatusCode, statusExpl)
+				assert.Contains(t, resp.Body, blockedMsg, bodyExpl)
+			})
+		}
```

Comment:
The `CAR` times out on both .26 and current PR. Not sure exactly why.

Reply:
It feels like some tests fail due to the same bug as https://github.com/ipfs/kubo/issues/10361